src/NUnitFramework/framework/Internal/StringUtil.cs

```diff
@@ -41,8 +41,10 @@ public class StringUtil
         /// strB is sorted first</returns>
         public static int Compare(string strA, string strB, bool ignoreCase)
         {
-            var comparison = ignoreCase ? StringComparison.CurrentCultureIgnoreCase : StringComparison.CurrentCulture;
-            return string.Compare(strA, strB, comparison);
+            string s1 = ignoreCase ? strA.ToUpperInvariant() : strA;
+            string s2 = ignoreCase ? strB.ToUpperInvariant() : strB;
+
+            return String.CompareOrdinal(s1, s2);
```

Comment:
Same [@Lette Looks like we want to keep the original behavior, so let's not make any changes at all. `string.Compare(,,)` would be an improvement but orthogonal to the purposes of this PR.]